data_services/deploy/ansible/roles/dataqs/files/geonode_patch.diff

```diff
@@ -0,0 +1,42 @@
+diff --git a/geonode/__init__.py b/geonode/__init__.py
```

Comment:
It is.  I'm using it to simplify the deployment.  Rather than inherit from the geonode app and go through the hassle of setting up a whole new app.  I just twiddle a few files from geonode to make it do what we want.